### PR TITLE
Unify return types in PageRequest.

### PR DESCRIPTION
**Pull request by Artur-**
- [X] You have read the [Spring Data contribution guidelines](https://github.com/spring-projects/spring-data-build/blob/master/CONTRIBUTING.adoc).
- [X] You use the code formatters provided [here](https://github.com/spring-projects/spring-data-build/tree/master/etc/ide) and have them applied to your changes. Don’t submit any formatting related changes.
- [X] You submit test cases (unit or integration tests) that back your changes.
- [X] You added yourself as author in the headers of the classes you touched. Amend the date range in the Apache license header if needed. For new types, add the license header (copy from another file and set the current year only).

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@Artur- Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-data-commons&pullRequestId=2318)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-projects/spring-data-commons&pullRequestId=2318) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@Artur- Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-data-commons&pullRequestId=2318)!

File: src/main/java/org/springframework/data/domain/PageRequest.java [hunk 1]
```diff
@@ -118,7 +119,7 @@ public Pageable next() {
 	 * @see org.springframework.data.domain.AbstractPageRequest#previous()
 	 */
 	@Override
-	public PageRequest previous() {
+	public Pageable previous() {
```
Comment (mp911de) [inline src/main/java/org/springframework/data/domain/PageRequest.java:122]: We should rather change the other methods to return `PageRequest` as that is the more concrete type and we retain type-safety. Care to have a look?

**Review by mp911de — CHANGES_REQUESTED** [review]

**Comment by mp911de** [comment]
Since there was no further activity we decided to handle the changes ourselves.

**Pull request closed without merging**